packages/server-boilerplate/src/orchestrator/routes/RequestResetPasswordRoute.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ *
+ */
+
+import { Request, Response, NextFunction } from 'express';
+import { AuthConnection, AuthResponse } from '../auth';
+import { Route } from '../../routes';
+import { EmptyObject } from '../../types';
+import { RequestResetPasswordCredentials } from '../types';
+
+export interface RequestResetPasswordRequest
+  extends Request<EmptyObject, AuthResponse, RequestResetPasswordCredentials> {
+  ctx: {
+    apiName?: string;
+  };
```

Comment:
```suggestion
```
Not relevant on this interface so it's nice to omit things to keep the types as simple as possible